src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/PerformerNameAndAddress.java

```diff
@@ -9,9 +9,11 @@
 import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
 
 /**
- * Example: NAD+SLA+A2442389:902++DR J SMITH'
+ * Example: NAD+SLA+A2442389:902++DR J SMITH' (Details of performer)
  * <br>
- * Example: NAD+SLA+++ST JAMES?'S UNIVERSITY HOSPITAL'
+ * Example: NAD+SLA+++Haematology' (Details of performing organization department)
+ * <br>
+ * Example: NAD+SLA+++ST JAMES?'S UNIVERSITY HOSPITAL' (Details of performing organization)
```

Comment:
Should we mention the SPR+... segment here in the examples, since its how department and organisation are distinguished?

Reply:
We have the `ServiceProvider` parser for that. But I'll reference it in the docs here